pinot-core/src/main/java/org/apache/pinot/core/data/manager/BaseTableDataManager.java

```diff
@@ -858,6 +868,9 @@ protected File downloadSegmentFromPeers(SegmentZKMetadata zkMetadata)
     _logger.info("Downloading segment: {} from peers", segmentName);
     File tempRootDir = getTmpSegmentDataDir("tmp-" + segmentName + "-" + UUID.randomUUID());
     File segmentTarFile = new File(tempRootDir, segmentName + TarCompressionUtils.TAR_COMPRESSED_FILE_EXTENSION);
+    if (_segmentOperationsThrottler != null) {
+      _segmentOperationsThrottler.getSegmentDownloadThrottler().acquire();
```

Comment:
Same here, add some info for debugging purpose

Reply:
done